urid/sys/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "lv2-urid-sys"
+version = "0.1.0"
+authors = ["Janonard <janonard@protonmail.com>"]
+edition = "2018"
+license = "MIT OR Apache-2.0"
+
+[dependencies]
+
+[build-dependencies]
+bindgen = "0.51.0"
```

Comment:
I do not think having a build-time dependency on bindgen is a good thing.
It is a very big dependency (hence increasing build times quite a lot), on top of requiring `clang` to be installed if I remember correctly.
Since LV2 is not a dynamic library but a standardized header file, the only thing it allows us `lv2-rs` developers is automating the bindings generation whenever we change the header file, but considering how often LV2 headers update (and since we vendored them in anyways), it is not something that I would mind doing manually. Hence I do not think it is a price worth paying for our users.

Maybe it would be better to do this the same way as `lv2-core-sys`, i.e. vendoring in the `bindings.rs` that been generated through the locally-installed bindgen CLI, with the exact command specified in a `CONTRIBUTING.md` file.

Reply:
I did add that because `bindgen` produces a different output on other platforms. For example, I tried to build and test the crate on my Raspberry Pi, but failed since the struct alignment is different there. Since you've stated in the [Readme](https://github.com/rust-dsp/rust-lv2/blob/master/README.md) that LV2 is cross-platform, we should also ensure that your crate is truly cross-platform! :wink: This is either possible by providing a different crate, including every platform as a module in the crate or by generating the bindings on-the-fly.

Since the first two options are quite tedious, error prone and probably incomplete, I've tried using bindgen in the build script. Compiling all tests in release mode from scratch takes roughly one minute on my machine, which is an average-speced notebook, and I think that's okay. However, we should add a note in the Readme that we require clang to be installed. Someone would have to do that.

Reply:
Oh, I did not realize the bindgen output was not cross-platform!

Then I assume we should better update `lv2-core-sys` to be automatically generated as well, we just have to lock bindgen to the same version to not exponentially increase compile times the more `-sys` crates we add.
But in general, I agree we shouldn't care about it increasing compile times if it prevents the crate from working on other plaftorms. ;)

It's not a big change I think, so I can do it if you don't mind (as well as updating the README).

Reply:
I've already implemented all of these things! :wink: